style.css

```
@@ -50,8 +50,9 @@ img, picture {
   --n-blue-dark: hsl(218, 23%, 16%); /* neutral: dark blue */
 
   /* fonts */
-  --fs-quote: 1.75rem; /* font size: 28px */
-  --fs-title: 1.75rem;
+  --fs-quote-d: 1.75rem; /* font size: 28px on dekstop */
+  --fd-quote-m: 1.5rem; /* font size: 24px on mobile */
```

```suggestion
  --fs-quote-m: 1.5rem; /* font size: 24px on mobile */
```
looks like a typo haha

---

eagle eye!

---

I was just stumped trying to figure out what fd stood for...